Core/Config.h

```diff
@@ -160,6 +160,7 @@ struct Config {
 	std::string sMicDevice;
 	bool bCameraMirrorHorizontal;
 	int iDisplayFramerateMode;  // enum DisplayFramerateMode. Android-only.
+	int iDisplayRefreshRate = 60;
```

Comment:
This initialization probably not needed anymore, just like the other vars in this struct.

Reply:
This would break the tests (double checked) since there's code reading this field before the config constructor runs and sets the user config/default. They expect to read 60.

This assignment is done at compile-time and stored in the data section, so it's all good.

See https://github.com/hrydgard/ppsspp/pull/19325/commits/1aec09a056b616034677f106e2450c8833a083b2 's description